decidim-core/app/cells/decidim/upload_modal/modal.erb

```diff
@@ -1,25 +1,24 @@
-<div class="reveal upload-modal" id="<%= modal_id %>" data-reveal>
-  <div class="reveal__header">
-    <h3 class="reveal__title" data-addlabel="<%= button_label %>" data-editlabel="<%= button_edit_label %>"><%= label %></h3>
-    <button class="close-button" data-close aria-label="<%= t("layouts.decidim.upload_modal.close_modal") %>"
-      type="button">
-      <span aria-hidden="true">&times;</span>
-    </button>
-  </div>
+<%= decidim_modal id: modal_id, class: "upload-modal" do %>
```

Comment:
> I cannot click the text "Drop files here or click to upload":

I think the quickest solution is about removing that part of the text, or be more specific: _click the button to upload_. The design includes a button inside, besides, it's wrapped using a block, therefore we faced several a11y issues regarding div inside labels.

Reply:
We're aware of that error, along the "If you load both jquery_ujs and rails-ujs, use rails-ujs only". They're come up after Turbo was introduced. Since that's a dependencies problem, it's not related with this issue, so we're facing it later on.